3_summarize/src/calculate_total_benthic_area.R

```diff
@@ -13,6 +13,12 @@ calculate_total_benthic_area <- function(target_name, all_hypsos) {
 }
 
 extract_hypsography <- function(morphometry) {
-  # Only keep depths (H) and areas (A)
-  purrr::map(morphometry, `[`, c("H", "A"))
+  # Only keep depths (converted from Heights, H) and areas (A)
+  purrr::map(morphometry,  function(m) {
+    hypsos <- data.frame(H = m$H, A = m$A) %>% 
+      mutate(depths = max(H) - H, areas = A) %>% 
```

Comment:
looks good

Reply:
Do you want to revert this change then since it would break the internal TOHA calculation (the thing you caught in your comment above)?

Reply:
revert which change?

Reply:
Wait, I think I see what you mean. The fxn I changed here is called `extract_hypsography` and is currently only used for the total benthic area step. It does not affect the TOHA stuff at all. However, maybe I could pull out the hypso into this step and pass that in differently to the TOHA.

Reply:
I am referring to the change here to `extract_hypsography`. If you don't revert it, you will error when `morphometry` gets passed into [calculate_toha_per_lake](https://github.com/USGS-R/lake-temperature-out/blob/master/2_process/src/calculate_toha.R#L9-L12) which will be looking for `H` and `A` and instead will get `depths` and `areas`. And if we alter `morphometry`, we'll have to rebuild all of the toha outputs. I figured you'd be reverting this change and moving the `H` to `depths` mutate into the `calculate_total_benthic_area` but perhaps you will keep the change here and then remove the translation that happens in `calculate_toha_per_lake`. 

Anyhow, let me know if this is still draft, as I think I am commenting on in progress code.

Reply:
`extract_hypsography` should not affect `calculate_toha_per_lake` at all since it is getting `morphometry` passed in and not `hypsography`.

Reply:
This might be easier to explain over the phone

Reply:
I see now, since 
```yaml
  nhdhr_120020218_morphometry:
    command: morphometry[[I('nhdhr_120020218')]]
```
looks good

Reply:
I get it now. I thought extract_hypsography was used to prep for toha. All good.

Reply:
I removed `extract_hypsography` altogether so it wouldn't be confusing.